sockets/1.2/modules/ROOT/pages/edits/index.adoc

```diff
@@ -0,0 +1,36 @@
+= Sockets Connector 1.2 - Mule 4
+:page-aliases: connectors::sockets/sockets-connector.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Sockets (Sockets Connector) provides a sockets listener, send capability, and send and receive functionality. Additionally, provides TCP and UDP requesters.
```

Comment:
```suggestion
Anypoint Connector for Sockets (Sockets Connector) provides a sockets listener, send capability, and send and receive functionality. Additionally, it provides TCP and UDP requesters.
```

Reply:
Do we need to say anything about WHY or under what circumstances our audience wants or needs these functionalities? Or is that just too obvious?